torch/nn/modules/module.py

```diff
@@ -85,6 +85,7 @@ def __init__(self):
         self.training = True
         self._parameters = OrderedDict()
         self._buffers = OrderedDict()
+        self._persistent_buffers = set()
```

Comment:
nit: to make this clearer that this is not a dict like the other things, you could name this `self._persistent_buffers_name_set` ?  Or is this getting too long?

Reply:
Done. I'm not a huge fan of appending the type name to the identifier, but don't feel strongly about it.